docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/enforcement.adoc

```diff
@@ -99,160 +105,119 @@ For every failed scan result you can view the latest Pull Request (PR) of your r
+Configuring an exception includes defining the scope of the exception, and specifying the repositories code categories and run rules that the exception will apply to.
 
 [.procedure]
 
-. <<access-enforcement,Access>> enforcement.
+. <<access-enforcement,Access Enforcement>> settings.
 
-. Add an exception to enforcement.
-.. Select *Add exception*.
+. Select *Add Exception*.
 +
 image::application-security/enfor-6.png[]
 
-. Configure exception parameters.
-.. Add *Description* to the new exception.
-.. Select the repositories you want to add the exception.
+. Add a *Description* to the new exception.
+. Select the repositories you want to add the exception.
 +
 NOTE: You can only view repositories that you own.
 
-.. Select a code category.
-
-.. Select the severity threshold corresponding to the code category.
-+
-You can choose to continue modifying other code categories or conclude with a single modification.
+. Modify a severity threshold corresponding to the required code category/ categories.
 
-.. Select *Save* to save the exception with the parameters.
+. Select *Save*.
 +
-Here is an example of an exception.
+EXAMPLE
 +
 image::application-security/enfor-21.png[]
 +
-All exception configurations are listed on *Enforcement*.
+NOTE: All exception configurations are listed on *Enforcement*.
 +
 image::application-security/enfor-22.png[]
-+
-You can optionally choose to edit or delete an existing exception.
-+
-** To edit an exception, hover over the Exception and then select *Edit* to configure the parameters. Select *Save* to save the modification to the exception.
-+
-image::application-security/enfor-10.png[]
-+
-** To delete an exception select *Edit* and then select *Delete this exception*.
-+
-image::application-security/enfor-11.png[]
 
-[.task]
+==== Editing/Deleting an Exception
 
-[#turn-off-run-rule-scan-for-a-code-category]
-=== Turn off run rule scan for a code category
+You can edit or delete exceptions.
 
-You can choose to turn off one or more run rules for code categories, if your enforcement strategy is aligned with it.
+* To edit an exception, hover over the Exception and then select *Edit* to configure the parameters. Select *Save* to save the modification to the exception.
 
-NOTE: Turning the scan off for a run rule in a code category results in no code review scan.
+* To delete an exception, select *Edit* and then select *Delete this exception*.
 
-[.procedure]
-. <<access-enforcement,Access>> Enforcement.
+=== Disable Enforcement Rules
 
-. Select a code category.
+* Global disable: You can disable enforcement rules entirely: uncheck the *Enable default thresholds for soft-fail, hard-fail and comments bot in your code reviews* setting
```

Comment:
This should be converted to a procedure and be rewritten as steps as we are providing instruction on how to disable enforcement rules.

Reply:
It is a single step